tests/types/src/valid.ts

```diff
@@ -870,7 +870,7 @@ expressCheckoutElement
     }) => {}
   );
```

Comment:
It might be worth adding validation for the click event here as well.

Reply:
I believe that's taken care of with `StripeExpressCheckoutElementClickEvent`!